src/main/java/Duke.java

```diff
@@ -1,10 +1,31 @@
+import duke.util.List;
+import duke.util.Parser;
+import duke.util.Ui;
+
 public class Duke {
+
+    private final Ui ui;
+    private final List list;
+    private final Parser parser;
+
+    /**
+     * Constructor of Duke.
+     */
+    public Duke() {
+        this.ui = new Ui();
+        this.list = new List();
+        this.parser = new Parser(this.list);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        new Duke().run();
+    }
+
+    /**
+     * Run the Duke program, prints out welcome statement and starts accepting input.
+     */
```

Comment:
Javadoc headers should start with words in the form `Returns ...`, `Sends ...`, `Adds ...` (not `Return` or `Returnning` etc.)
This applies to your other javadoc headers in your other classes too! I believe I spotted some that do not follow this convention. 😬

Reply:
I agree with @BryannYeap about javadocs. Most of your javadocs headers have the same problem.:smile:

Reply:
haha yea thanks for pointing out

Reply:
Seconded.